sgkit/stats/hwe.py

```diff
@@ -156,8 +156,9 @@ def hardy_weinberg_test(
     Raises
     ------
     NotImplementedError
-        * If ploidy of provided dataset != 2
-        * If maximum number of alleles in provided dataset != 2
+        If ploidy of provided dataset != 2
+    NotImplementedError
```

Comment:
I had originally been breaking multiple causes of the same error out like this but @ravwojdyla suggested collapsing them in a past PR.  @ravwojdyla did you have a reason for that?

Reply:
Sphinx doesn't render the bullets correctly, which is why I changed this.